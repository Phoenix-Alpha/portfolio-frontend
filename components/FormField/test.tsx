import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { FormField } from '.'

describe('<FormField />', () => {
  const name = 'test-name'

  it('should renders correct <FormField />', () => {
    const wrapper = shallow(<FormField name={name} />)

    expect(wrapper.find('Field').prop('name')).toMatch(name)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
