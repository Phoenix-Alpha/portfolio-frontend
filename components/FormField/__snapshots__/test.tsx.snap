// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormField /> should renders correct <FormField /> 1`] = `
<Field
  name="test-name"
>
  <Component />
</Field>
`;
